mne/decoding/transformer.py

```diff
@@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
 class PSDEstimator(TransformerMixin):
     """Compute power spectrum density (PSD) using a multi-taper method
 
+    This structures data so that it can be easily incorporated into
+    scikit-learn pipelines. It relies heavily on the multitaper_psd
+    function found in the time_frequency module. Running the transform
+    method will create attributes for the estimated psd as well as the
```

Comment:
I believe I still do at the very end, I just only add a freqs_ attribute and not a data_ attribute per @agramfort 's last comment